doc/decisions/1_in_discussion/change_tracking.md

```diff
@@ -41,68 +41,62 @@ Change tracking must:
 - if overhead is not negligible: only do tracking as required, i.e., a plugin specifically requests it
 - have negligible overhead if disabled
 - not duplicate data for each plugin that wants change tracking
-- work with all allowed sequences of `kdbGet` and `kdbSet` as [per this decision](operation_sequences.md)
+- work with all allowed sequences of `kdbGet` and `kdbSet` as [per this decision](../0_drafts/operation_sequences.md)
+
+We only want to track changes that are done by the user, not changes that are done by plugins.
+I.e. the scope of change tracking is on what happens _outside_ of `kdbGet` and `kdbSet`.
 
 The library `libelektra-core` must be kept minimal.
 
 ## Assumptions
 
 - It is possible to do change tracking with reasonable memory and computation overhead
-- It is possible to design a single change tracking API that is useful
-  for all existing and future plugins
+- It is possible to design a single change tracking API that is useful for all existing and future plugins
+- False positivies are okay
 
-## Considered Alternatives
+  - this may happend when some keys may have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
```

Comment:
```suggestion
  - this may happend when some keys have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
```